Windows/CMakeLists.txt

```diff
@@ -36,4 +35,8 @@ IF (MSVC)
     add_test(NAME daemon_windows_tests COMMAND ${PROJECT_BINARY_DIR}/bin/daemon_windows_tests.exe)
 ELSE ()
     add_test(NAME daemon_windows_tests COMMAND daemon_windows_tests)
-ENDIF ()
\ No newline at end of file
+ENDIF ()
+
+####INSTALL####
+
+install(TARGETS daemon_windows DESTINATION bin)
```

Comment:
💩 It's useless. No ?

Reply:
Probably yes, but the idea is that the packaging step leverages the install commands to build an NSIS installer via CPack. For now, there is no packaging on Windows yet.